plugins/package-managers/maven/src/main/kotlin/utils/LocalRepositoryHelper.kt

```diff
@@ -43,6 +48,44 @@ class LocalRepositoryHelper(
 
         /** The name of the folder that stores OSGi bundles. */
         private const val BUNDLE_ROOT = "osgi/bundle"
+
+        /** The name of the XML file with properties for P2 artifacts contained in the local repository. */
+        private const val ARTIFACTS_FILE = "p2artifacts.xml"
```

Comment:
Commit message:
- "ending on" -> "ending with" (But is that really only the file ending, or the whole name?)

Reply:
It is indeed a suffix that is appended to the file name of the `jar` artifact. In the commit message, I have added the separator "-" to the suffix to make this clearer.

Reply:
But shouldn't then both the code comment and variable name be adjusted? "The name of the XML file" should probably say "The suffix of ...", and similar `ARTIFACTS_FILE` is not a file, but a filename, and also of that only the suffix.